src/users/users.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@nestjs/common';
+import { User } from './interfaces/user.interface';
+
+@Injectable()
+export class UsersService {
+    private users: Map<number, User> = new Map();
+    private userIDcounter: number = 0;
```

Comment:
:wrench: Let's add a little comment for the future, that this needs to be changed again in the future implementation.

```suggestion
    //TODO: #26 - remove users and userIDcounter when implementing UserService properly
    private users: Map<number, User> = new Map();
    private userIDcounter: number = 0;
```